crml/staking/src/lib.rs

```diff
@@ -814,8 +814,6 @@ decl_error! {
 		NotSortedAndUnique,
 		/// Cannot nominate the same account multiple times
 		DuplicateNominee,
-		// TODO remove the following error when https://github.com/cennznet/cennznet/issues/297
-		BondingNotEnabled,
```

Comment:
Didn't we want to use the sudo command to enable bonding?

Reply:
it's already live on mainnet, it won't be needed after this

Reply:
So please remove the corresponding storage as well:
`BlockBonding get(fn block_bonding) config(): bool;`
In fact, we should kill the storage in the runtime upgrade as well. I can actually do that in https://github.com/cennznet/cennznet/pull/368 if you prefer.

Reply:
Now that https://github.com/cennznet/cennznet/pull/368 is merged, can you also please do `inner::BlockBonding::kill();` after merging develop into your branch?